platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/scheduled/AGScheduledProgramExecution.sqml

```diff
@@ -8,6 +8,8 @@ TABLE AGScheduledProgramExecution 'Core.ScheduledProgramExecution' {
 	Boolean active = TRUE
 	String cronExpression [MAXLENGTH=255]
 	AGUuid programUuid
+	Integer maximumRuntime = NULL [NULLABLE]
+	Boolean autoKill = FALSE
```

Comment:
I think we need a validator to ensure that a `maximumRuntime` is defined when `autoKill` is true.